xapian-letor/bin/questletor.cc

```diff
@@ -42,16 +43,8 @@
 
 using namespace std;
 
-#define PROG_NAME "letor"
-#define PROG_DESC "Xapian command line search tool with Lerning to Rank Facility"
-
-typedef std::pair<Xapian::docid, double> MyPair;
-
-struct MyTestCompare {
-    bool operator()(const MyPair& firstPair, const MyPair& secondPair) const {
-	return firstPair.second < secondPair.second;
-    }
-};
+#define PROG_NAME "letor-rank"
```

Comment:
If its help text is going to say this, it should probably be renamed to that. (Or pull the `PROG_NAME` out of `argv[0]`?)